packages/odyssey-react/src/components/Modal/index.tsx

```diff
@@ -65,8 +66,9 @@ export type PropsModalFooter = {
 
 export type PropsModalButton = {
   close?: boolean;
-  children: ReactText;
+  children?: ReactText;
   variant?: ButtonProps["variant"];
+  icon?: ReactElement;
```

Comment:
Why pass this as a prop? Why not just bake the close icon into the button? It should never change

Reply:
Looking at this again, I think this is my fault. I implemented it this way 😂, my opinion is that the icon should be baked in to the button at all times.